### PR TITLE
[FEATURE] add pull to refresh feature

### PR DESCRIPTION
**Pull request by sarahmarciano**
add the pull to refresh feature to the tickets tab, demands tab and approvals tabs
(there are some things that gonna change once we will refactor the tickets component such as we will have just one ticket list component and don't have duplicate code as we have right now)
screenshot_1508940619

### CODE REVIEW

**Comment by philjoseph** [comment]
@raphaelboukara Did you review this one ?

File: js/actions/ApprovalsActions.js [hunk 1]
```diff
@@ -50,3 +51,19 @@ export const setApproval = (accept, ticketId, comment) => {
         });
     };
 };
+
+export const refreshApprovals = () => {
+    return (dispatch) => {
```
Comment (raphaelboukara) [inline js/actions/ApprovalsActions.js]: why you have to wrap this in a function?

File: js/actions/DemandsActions.js [hunk 2]
```diff
@@ -17,3 +18,14 @@ export const getDemands = () => {
         }
     };
 };
+
+export const refreshDemands = () => {
+    return (dispatch) => {
```
Comment (raphaelboukara) [inline js/actions/DemandsActions.js]: same question: is the promise middleware you are using works like this?

File: js/actions/TicketsActions.js [hunk 3]
```diff
@@ -17,3 +18,14 @@ export const getTickets = () => {
         }
     };
 };
+
+export const refreshTickets = () => {
```
Comment (raphaelboukara) [inline js/actions/TicketsActions.js]: same

**Pull request merged**